### DISCUSSION TITLE
Backups are in UTC time, not timezone of clickhouse.

### DISCUSSION THREAD

**Discussion by bunkiedc** [Altinity/clickhouse-backup#471] [Q&A] [answered]
My clickhouse db is set to my local timezone, but the backups are being saved to catalog with UTC.  
Is there a way of changing that so that these are also in my local timezone?

**Comment by Slach** [comment]
Do you mean you have `DateTime` columns in your data
And your destination server have UTC timezone, which different from your source server?
And, after restore, these columns showed in UTC time zone?

**Comment by bunkiedc** [comment]
No. I have clickhouse timezone set to Los_Angeles, my servers set to Los_Angeles.  When clickhouse-backup creates the backup, the directory name, the timestamp on the directory name is UTC time.  The directory name looks like this: 2022-07-13T14-35-00, which is UTC time, not Los_Angeles.

**Comment by Slach** [comment, marked as answer]
it's expected behavior
you can use something like `clickhouse-backup create $(date +%Y-%m-%dT%H:%M:%S)` if you want to custom name format

check
`clickhouse-backup help create`
for  detauls
